src/test/ui/feature-gate-try_reserve.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn main() {
+    let v = Vec::new();
+    v.try_reserve(10); //~ ERROR: attribute is an experimental feature
```

Comment:
You need to adjust the error message there to what the compiler actually outputs. `use of unstable library feature "try_reserve" [...]` or so.

Reply:
Thanks, solved!